packages/expo-updates/android/src/main/java/expo/modules/updates/loader/EmbeddedLoader.kt

```diff
@@ -99,26 +93,7 @@ class EmbeddedLoader internal constructor(
   }
 
   override fun shouldSkipAsset(assetEntity: AssetEntity): Boolean {
```

Comment:
I think we can remove this function and `AssetLoadResult.SKIPPED` entirely?

Reply:
We have to implement that function in the abstract class.

Reply:
Sorry my review wasn't clearer, I meant we can remove it from the abstract class as this was the only use of it.

Reply:
Sigh yes I just noticed that you beat me to it 😄 . Done.